python/rmm/rmm/librmm/_logger.pxd

```diff
@@ -16,7 +16,7 @@ from libcpp cimport bool
 from libcpp.string cimport string
 
 
-cdef extern from "rmm/logger.hpp" namespace "rmm" nogil:
+cdef extern from "rapids_logger/logger.hpp" namespace "rapids_logger" nogil:
```

Comment:
@vyasr Is this the part where you were concerned about re-implementing Cython pxd's for `rapids_logger` for multiple libraries? What was the concern again? Should we ship a pxd with rapids-logger, or is it C++-only?

Reply:
Yes, exactly. Right now every repo that needs to export a logger to Python needs to redeclare rapids_logger types. It turns out that is only two places though, rmm and cuml. Neither cudf nor raft expose their logger type directly to Python, so I'm OK punting on this improvement until a later date. As I mentioned when we discussed before, changing it now would probably introduce more Python packaging scaffolding than we would save in reducing pxd duplication.

Reply:
Is this worth creating an issue for?

Reply:
Agreed. We should document this in a rapids-logger issue just so we have something to link to when this inevitably comes up again. The Slack discussion was useful, but something publicly linked would be good.

Reply:
https://github.com/rapidsai/rapids-logger/issues/23